src/cli/cli-report.c

```diff
@@ -796,8 +796,10 @@ static int run_event_chain_real(struct run_event_state *run_state,
             break;
         if (retval == 0 && run_state->children_count == 0)
         {
-            printf("Error: no processing is specified for event '%s'\n", event_name);
-            retval = 1;
+            /* Continue running the chain even if no actions were executed for
+             * this event. */
+            log_warning("no commands were run for event '%s'", event_name);
```

Comment:
No commands run or none specified?

Reply:
None were run because no actions matching the problem under consideration were found. I'll reword the message to make that clearer.